action/JointModeCommand.action

```diff
@@ -0,0 +1,30 @@
+# Used in time-stamping the goal.
+Header header
+
+# Name list of the joints. You don't need to specify all joints of the
+# robot. Joint names are case-sensitive.
+string[] joint_names
+
+# Mode command to the joints listed in joint_names.
+# The order must be identical.
+int32[] modes
+
+int32 BEGIN          = -1
+int32 POSITION       = 0
+int32 VELOCITY       = 1
+int32 EFFORT         = 2
+int32 NOMODE         = 3
+int32 EMERGENCY_STOP = 4
+int32 SWITCHING      = 5
+int32 ERROR          = 6
+int32 OTHER          = 3
```

Comment:
Actually this is a small thing so feel free to ignore it if you want, but can you move OTHER to the line right after NOMODE (line 17)? At first when I read this I thought OTHER was a distinct mode, until I looked at this [inter-dependent PR](https://github.com/personalrobotics/pr_ros_controllers/pull/27) and realized that OTHER and NOMODE are the same.